Tools/autotest/sim_vehicle.py

```diff
@@ -493,7 +501,7 @@ def options_for_frame(frame, vehicle, opts):
     if frame in _options_for_frame:
         ret = _options_for_frame[frame]
     else:
-        for p in ["octa", "tri", "y6", "firefly", "heli", "last_letter", "jsbsim", "quadplane", "plane-elevon", "plane-vtail", "plane"]:
+        for p in ["octa", "tri", "y6", "firefly", "heli", "gazebo", "last_letter", "jsbsim", "quadplane", "plane-elevon", "plane-vtail", "plane"]:
```

Comment:
I am not sure if it's necessary to keep the old format (`Gazebo`). Here's a cleaner patch I can merge if you think it's OK:

~~~
diff --git a/Tools/autotest/sim_vehicle.py b/Tools/autotest/sim_vehicle.py
index 0ec48b6..1e304bf 100755
--- a/Tools/autotest/sim_vehicle.py
+++ b/Tools/autotest/sim_vehicle.py
@@ -457,16 +457,12 @@ _options_for_frame = {
         "waf_target": "bin/ardurover",
         "default_params_filename": "default_params/rover-skid.parm",
     },
-    # SIM (use GazeboIris in place of Gazebo)
-    "Gazebo": {
+    # SIM
+    "gazebo-iris": {
         "waf_target": "bin/arducopter-quad",
         "default_params_filename": "default_params/gazebo-iris.parm",
     },
-    "GazeboIris": {
-        "waf_target": "bin/arducopter-quad",
-        "default_params_filename": "default_params/gazebo-iris.parm",
-    },
-    "GazeboZephyr": {
+    "gazebo-zephyr": {
         "waf_target": "bin/arduplane",
         "default_params_filename": "default_params/gazebo-zephyr.parm",
     },
~~~